hebiros_gazebo_plugin/include/hebiros_gazebo_pid.cpp

```diff
@@ -0,0 +1,33 @@
+#include "hebiros_gazebo_pid.h"
+
+namespace hebiros {
+namespace sim {
+
+// Update and return the new output command
+double PidController::update(double target, double feedback, double dt, const PidGainsMsg& pid_gains, size_t gain_idx)
+{
+  // "Disable" the controller if commands are nan
+  if (std::isnan(target)) {
+    return 0;
+  }
+  double error_p, error_i, error_d;
+  error_p = target - feedback;
+  error_i = _elapsed_error + error_p;
+  error_d = (error_p - _prev_error) / dt;
+  _prev_error = error_p;
+  _elapsed_error = error_i;
+
+  if (dt <= 0)
```

Comment:
can `dt` be negative? If so, you should change this to `if (dt <= -0.0)` since `-0.0 != 0.0` unfortunately

Reply:
nvm I'm dumb